micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientIntegrationSpec.groovy

```diff
@@ -75,6 +77,23 @@ class ServiceRestClientIntegrationSpec extends MvcWiremockIntegrationSpec {
             exception.message.contains('Read timed out')
     }
 
+    def "should be interrupted and throw UncheckedTimeoutException when service does not respond"() {
+        given:
+            Integer fixedResponseDelayMilliseconds = readTimeoutMillis / 2
+            Integer circuitBreakersTimeoutInMillis = fixedResponseDelayMilliseconds / 2
+            String delayedPath = "/delayed-for-interruption"
+            stubInteraction(wireMockGet(delayedPath), aResponse().withFixedDelay(fixedResponseDelayMilliseconds))
```

Comment:
In this place delayed value is depend on rest.client.readTimeout and I think it's enough